src/plugins/intel_npu/src/common/include/intel_npu/common/icompiled_model.hpp

```diff
@@ -7,6 +7,7 @@
 #include <memory>
 #include <vector>
 
+#include "intel_npu/common/igraph.hpp"
 #include "intel_npu/config/common.hpp"
 #include "intel_npu/icompiler.hpp"
```

Comment:
This can be removed now. That was the objective.

Reply:
done, thx